manifests/mod/disk_cache.pp

```diff
@@ -1,11 +1,18 @@
 class apache2::mod::disk_cache (
   $activated = true) {
-  apache2::module { 'disk_cache':
-    activated          => $activated,
-    conf_file          => true,
-    conf_file_template => true,
-    require            => [
-      Package['httpd'],
-      Class['apache2::mod::cache']],
+
+  case $apache2::params::apache_version {
+    /(2.2)/ : {
```

Comment:
should it be 2.4?

Reply:
nope, the disk_cache module was renamed cache_disk in apache 2.4 so I add this to avoid to install it transparently and avoid apache to start later.